### PR TITLE
Misc test updates

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Flyout/Flyout_ShowAt_Window_Content.xaml.cs [hunk 1]
```diff
@@ -13,9 +15,20 @@ namespace UITests.Windows_UI_Xaml_Controls.FlyoutTests
...
 	{
+		private XamlRoot _xamlRoot;
+
 		public Flyout_ShowAt_Window_Content()
 		{
 			this.InitializeComponent();
+
+#if HAS_UNO
+			Unloaded += (s, e) => _xamlRoot = XamlRoot;
+			Unloaded += (s, e) =>
```
Comment (ajpinedam) [inline src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Flyout/Flyout_ShowAt_Window_Content.xaml.cs:26]: Was the intention to subscribe to the same event twice with different actions?

Comment (jeromelaban) [reply src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Flyout/Flyout_ShowAt_Window_Content.xaml.cs:25]: Indeed, thanks!

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.4

**Pull request merged**